airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx

```diff
@@ -97,6 +98,11 @@ export const AuthenticationProvider: React.FC = ({ children }) => {
       news: userData.news ?? false,
     });
 
+    const previousAnonymousId = Cookies.get("ajs_anonymous_id");
+    if (previousAnonymousId) {
```

Comment:
Check if the cookie exists and if so, set the anonymousId instead of creating a new one